packages/client/hmi-client/src/components/documents/Document.vue

```diff
@@ -167,82 +141,71 @@
+						</header>
 					</template>
-					<div class="constrain-width">
-						<ul>
-							<li
-								class="github-link extracted-item"
-								v-for="(url, index) in githubUrls"
-								:key="index"
-							>
-								<import-code-button v-if="isEditable" :urlString="url" @open-code="openCode" />
-								<a :href="url" rel="noreferrer noopener">{{ url }}</a>
-							</li>
-						</ul>
-					</div>
+					<ul>
+						<li class="extracted-item" v-for="(url, index) in githubUrls" :key="index">
+							<import-code-button v-if="isEditable" :urlString="url" @open-code="openCode" />
+						</li>
+					</ul>
 				</AccordionTab>
-				<AccordionTab v-if="!isEmpty(urlArtifacts)">
+				<AccordionTab v-if="!isEmpty(otherUrls)">
 					<template #header>
-						Other URLs<span class="artifact-amount" id="OtherURLs"
-							>({{ urlArtifacts.length }})</span
-						>
+						<header id="Other-URLs">
+							Other URLs<span class="artifact-amount">({{ otherUrls.length }})</span>
+						</header>
 					</template>
-					<div class="constrain-width">
-						<ul>
-							<li v-for="ex in urlArtifacts" :key="ex.url" class="extracted-item">
-								<b>{{ ex.resourceTitle }}</b>
-								<div>
-									<a :href="ex.url" rel="noreferrer noopener">{{ ex.url }}</a>
-								</div>
-							</li>
-						</ul>
-					</div>
+					<ul>
+						<li v-for="ex in otherUrls" :key="ex.url" class="extracted-item">
+							<b>{{ ex.resourceTitle }}</b>
+							<div>
+								<a :href="ex.url" rel="noreferrer noopener">{{ ex.url }}</a>
+							</div>
+						</li>
+					</ul>
 				</AccordionTab>
-				<AccordionTab v-if="!isEmpty(otherArtifacts)">
+				<AccordionTab v-if="!isEmpty(otherExtractions)">
 					<template #header>
-						Other extractions<span class="artifact-amount" id="OtherExtractions"
-							>({{ otherArtifacts.length }})</span
-						>
+						<header id="Other-Extractions">
+							Other extractions<span class="artifact-amount">({{ otherExtractions.length }})</span>
+						</header>
 					</template>
-					<div class="constrain-width">
-						<div v-for="ex in otherArtifacts" :key="ex.askemId" class="extracted-item">
+					<ul>
+						<li v-for="ex in otherExtractions" :key="ex.askemId" class="extracted-item">
 							<b v-html="highlightSearchTerms(ex.properties.title)" />
 							<span v-html="highlightSearchTerms(ex.properties.caption)" />
 							<span v-html="highlightSearchTerms(ex.properties.abstractText)" />
 							<span v-html="highlightSearchTerms(ex.properties.contentText)" />
-						</div>
-					</div>
+						</li>
+					</ul>
 				</AccordionTab>
 				<AccordionTab v-if="!isEmpty(doc.citationList)">
 					<template #header>
-						References<span class="artifact-amount" id="References"
-							>({{ doc.citationList.length }})</span
-						>
+						<header id="References">
+							References<span class="artifact-amount">({{ doc.citationList.length }})</span>
+						</header>
```

Comment:
I thought it would be better to keep the formatting consistent within the Accordion Tabs. Before the ids were attached to the `<span class="artifact-amount" id='...'>`. Headers that had no need for `artifact-amount` were wrapped in a `<span id='...'>` since I had to put the id in the HTML.

Reply:
Fair enough, so it's about the table of content again. Gotcha!